cli/src/pcluster/resources/imagebuilder/parallelcluster_validate.yaml

```diff
@@ -167,6 +167,36 @@ phases:
               VERSION=$(echo ${PATTERN} | tr -d '\n' | cut -d = -f 2 | xargs)
               echo ${VERSION}
 
+      - name: CudaSamplesSrcDir
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              cuda_ver="{{ validate.CudaVersion.outputs.stdout }}"
+              if [ ${cuda_ver} \> '11.4' ]; then
```

Comment:
Mind that this is a string comparison, and `cuda_ver=011.5` will fail the test. It's OK if we are confident about the versioning format

Reply:
This will be retrieved from 'attributes/default.rb' in the cookbook recipes, so it is under our control. I can add some more processing to make sure it's normalized though if you like?

Reply:
So the sample download will come from github and is done as part of my yet to be generated pull-request for the cookbook package. I'm just trying to set up the test here so that it will work with either the embedded samples in 11.4 or the external samples in 11.7 - or at least that is my goal. Let me know if there is a better way to do this.

Reply:
If the value is under our control I'm good with this, tnx